tests/test_ec2/test_elastic_block_store.py

```diff
@@ -116,67 +120,69 @@ def test_volume_filters():
 
     block_mapping = instance.block_device_mapping['/dev/sda1']
 
+    volume_ids = (volume1.id, volume2.id, volume3.id, volume4.id, block_mapping.volume_id)
+
     volumes_by_attach_time = conn.get_all_volumes(
         filters={'attachment.attach-time': block_mapping.attach_time})
     set([vol.id for vol in volumes_by_attach_time]
-        ).should.equal(set([block_mapping.volume_id]))
+        ).should.equal({block_mapping.volume_id})
```

Comment:
I didn't know about this set notation. TIL!

Reply:
Yeah I normally use it unless I'm creating an empty set